assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -147,4 +147,60 @@
 			margin-bottom: 0;
 		}
 	}
+
+	.googlesitekit-ad-blocking-recovery__step {
+
+		.googlesitekit-stepper__step-content {
+
+			p {
+				color: $c-surfaces-on-surface-variant;
+				font-size: $fs-body-lg;
+				letter-spacing: $ls-s;
+			}
+
+			p:first-child {
+				margin-top: 8px;
+			}
+		}
+	}
+
+	.googlesitekit-ad-blocking-recovery__step-place-tags {
+
+		.mdc-checkbox {
+			margin-left: -11px;
```

Comment:
This is a bit odd however I see the same here https://github.com/google/site-kit-wp/blob/471e3665756f1d5589a71d45e471e19c5a8fd7cd/assets/sass/components/global/_googlesitekit-view-only-menu.scss#L119-L121

Are we missing this style on the checkbox itself? If not, but we need this in some cases and not others, it should probably be handled by a class to avoid duplicating this every time.

Reply:
I have addressed this in a slightly big commit - https://github.com/google/site-kit-wp/pull/7183/commits/97561d2f1707060439a70b452c6854bf1cdaa293.

To summarise, I have introduced a new prop to the `Checkbox` and `Radio` components - `alignLeft`, which under the hood applies the negative margin to compensate for the additional padding of the components. I have also replaced the existing negative margin usages to use this prop instead.

Please let me know if this is acceptable. Thank you!